ee/ucp/admin/configure/enable-audit-logging.md

```diff
@@ -1,99 +1,95 @@
 ---
 title: Enable audit logging on UCP
-description: Learn how to enable audit logging of all activity in UCP
+description: Learn how to enable audit logging of all activity in UCP.
 keywords: logs, ucp, swarm, kubernetes, audits
 redirect_from: /ee/ucp/admin/configure/create-audit-logs/
 ---
 
 >{% include enterprise_label_shortform.md %}
 
 Audit logs are a chronological record of security-relevant activities by 
-individual users, administrators or software components that have affected the 
+individual users, administrators, or software components that have affected the 
 system. They are focused on external user/agent actions and security rather than 
 understanding state or events of the system itself.
 
 Audit logs capture all HTTP actions (GET, PUT, POST, PATCH, DELETE) to all UCP 
-API, Swarm API and Kubernetes API endpoints that are invoked (except for the 
-ignored list) and sent to Docker Engine via stdout. Creating audit logs is a UCP 
-component that integrates with Swarm, Kubernetes, and UCP APIs.
+API, Swarm API, and Kubernetes API endpoints that are invoked (except for the 
+ignored list) and sent to Docker Engine through standard output (stdout). Creating audit logs is a UCP component that integrates with Swarm, Kubernetes, and UCP APIs.
 
 ## Logging levels
 
-To allow more control to administrators over the audit logging, three audit 
-logging levels are provided:
+The following levels are provided to give administrators more control over audit logging:
 
-- **None**: audit logging is disabled
+- **None:** Audit logging is disabled.
 
-- **Metadata**: includes the following:
+- **Metadata:** Includes the following:
     - Method and API endpoint for the request
     - UCP user who made the request
     - Response Status (success or failure)
     - Timestamp of the call
     - Object ID of any created or updated resource (for create or update API 
-    calls). We do not include names of created or updated resources
+    calls). The names of created or updated resources are not included.
     - License Key
     - Remote Address
 
-- **Request**: includes all fields from the Metadata level as well as the 
+- **Request:** Includes all fields from the Metadata level as well as the 
 request payload.
 
 > Note
 >
 > Once UCP audit logging has been enabled, audit logs can be found within the 
 > container logs of the `ucp-controller` container on each UCP manager node. 
-> Please ensure you have a 
-> [logging driver](../../../../config/containers/logging/configure/) 
-> configured appropriately with log rotation set as audit logging can start to 
-> generate a lot of data. 
+> Ensure that you have a [logging  driver](../../../../config/containers/logging/configure/) 
+> configured appropriately with log rotation set, as audit logging can start to 
+> generate a large amount of data. 
 
 ## Benefits
 
 You can use audit logs to help with the following use cases:
```

Comment:
Suggest editing the sentence to read: "Audit logs can be of help with the following use cases:"